ApexOSA/osafs.py

```
@@ -0,0 +1,333 @@
+        if TraceNumber < 0 or TraceNumber > 6:
+            raise ApexError(APXXXX_ERROR_ARGUMENT_VALUE, "TraceNumber")
+            sys.exit()
+
+        if self.__Simulation:
+            NPoints = randint(400, 600)
+        else:
+            Command = "OSAFSPOINTS" + str(int(TraceNumber)) + "\n"
+            Send(self.__Connexion, Command)
+            try:
+                NPoints = int(Receive(self.__Connexion)[:-1])
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)